schemas/choice.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://model.choice.json",
+  "title": "CHOICE",
+  "type": "object",
+  "properties": {
+    "locationNodeId": {
```

Comment:
I do not think we should have this pointer here. Since choice will be embedded in the node this pointer is redundant. Also this pointer makes it impossible to reuse same choice in multiple nodes.

Reply:
+